client/state/first-view/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function isViewVisible( state, view ) {
+	return -1 !== state.firstView.visible.indexOf( view );
+}
+
+export function isViewDismissed( state, view ) {
```

Comment:
Should be changed to `isViewEnabled` (note, not `isViewDisabled` -- find it easier to understand when using the positive state).